.prettierrc

```diff
@@ -1,5 +1,13 @@
 {
   "trailingComma": "es5",
   "singleQuote": true,
-  "printWidth": 80
-}
+  "printWidth": 80,
+  "overrides": [
+    {
+      "files": "*.html",
+      "options": {
+        "parser": "html"
+      }
+    }
+  ]
```

Comment:
What's the purpose of this?

Reply:
This was added in order to diffrentiate parsing for prettier in extension vs cli, however this ended up not fixing issue. I forgot to remove it as it had no effect, will remove this now. Thank you for catching this!